modules/ROOT/pages/choosing-the-right-clustering-topology.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: deploy, cloudhub, on premises, on premise, clusters
 
-You can deploy Mule in many different topologies. As you build your Mule application, it is important to think critically about how best to architect your application to achieve the desired availability, fault tolerance, and performance characteristics. This page outlines some of the solutions for achieving the right blend of these characteristics through clustering. There is no one correct approach for everyone, and designing your system is both an art and a science. If you need more assistance, MuleSoft Professional Services can help you by reviewing your architecture plan or designing it for you. For more information, http://www.mulesoft.com/contact[contact us].
+You can deploy Mule in many different topologies. As you build your Mule application, it is important to think critically about how best to architect your application to achieve the desired availability, fault tolerance, and performance characteristics. This page outlines some of the solutions for achieving the right blend of these characteristics through clustering when you deploy applications on-premises. There is no one correct approach for everyone, and designing your system is both an art and a science. If you need more assistance, MuleSoft Professional Services can help you by reviewing your architecture plan or designing it for you. For more information, http://www.mulesoft.com/contact[contact us].
```

Comment:
```suggestion
You can deploy Mule in many different topologies. As you build your Mule application, it is important to think critically about how best to architect your application to achieve the desired availability, fault tolerance, and performance characteristics. This page outlines some of the solutions for achieving the right blend of these characteristics through clustering when you deploy applications on premises. There is no one correct approach for everyone, and designing your system is both an art and a science. If you need more assistance, MuleSoft Professional Services can help you by reviewing your architecture plan or designing it for you. For more information, http://www.mulesoft.com/contact[contact us].
```

Reply:
dictionary says it's two words in cases like this: https://www.merriam-webster.com/dictionary/on%20premises